src/other/index.jl

```diff
@@ -131,6 +142,15 @@ function add_names(ind::Index, add_ind::Index)
         name = u[i]
         in(name, seen) ? push!(dups, i) : push!(seen, name)
     end
+    if length(dups) > 0
+        if makeunique
+            Base.depwarn("Keyword makeunique will be false by default.", :add_names)
```

Comment:
`makeunique` should default to `false` here, and the deprecation should only be printed when it's `false`. Else people have no way to silence these warnings. Same below.

Reply:
fixed - although I was not sure if what I did is what you have intended.

Reply:
Yeah, but the message should be changed to something like `Duplicate variable names are deprecated: pass makeunique=true to add a suffix automatically`.

Reply:
fixed